cmd/quick_shell/testnet.conf

```diff
@@ -0,0 +1,2 @@
+ip_port=""
+public_key=""
```

Comment:
is public_key referring to compliance service's address?

Reply:
Yes.